components/compliance-service/ingest/projectupdater/manager_test.go

```diff
@@ -141,12 +141,12 @@ func TestManagerMergeJobStatus(t *testing.T) {
 		{
 			description: "Two incomplete Jobs; use the job with the latest estimated complete date; opposite order",
 			jobStatuses: []ingestic.JobStatus{
-				ingestic.JobStatus{
+				{
 					Completed:             false,
 					PercentageComplete:    0.8,
 					EstimatedEndTimeInSec: 1555520534 + 20,
 				},
-				ingestic.JobStatus{
+				{
```

Comment:
These changes are the result of running `gofmt -s -w -l .` on compliance-service.